server/src/routes/login.ts

```diff
@@ -65,7 +65,9 @@ export async function authLogin(req: Request, res: Response, next: NextFunction)
               role: loginResult.role,
               id: loginResult.id,
             };
-            const token: string = jwt.sign(body, jwtSecret);
+            const token: string = jwt.sign(body, jwtSecret, {
+              expiresIn: '25h',
```

Comment:
Would it be better to set the `expiresIn` at `configs/config.ts`, now any changes must be applied to multiple places (here and tests)